paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -264,12 +276,10 @@ def paasta_mark_for_deployment(args):
                         commit=old_git_sha,
                     )
             else:
-                paasta_print("Waiting for deployment aborted. PaaSTA will continue to try to deploy this code.")
-                paasta_print("If you wish to see the status, run:")
-                paasta_print()
-                paasta_print("    paasta status -s %s -v" % service)
-                paasta_print()
+                report_waiting_aborted(service, args.deploy_group)
             ret = 1
+        except NoSuchCluster:
```

Comment:
Can you print "why" it aborted? or it is already covered when the exception is raised?
Does it print which cluster it not defined?